float64 goal_x
float64 goal_y
float64 threshold
---
float64 desired_angle
float64 h_prim_c